docs/src/main/sphinx/admin/dynamic-filtering.rst

```diff
@@ -3,35 +3,44 @@ Dynamic filtering
 =================
 
 Dynamic filtering optimizations significantly improve the performance of queries
-with selective joins by avoiding reading of data that would be filtered by join condition.
+with selective joins by avoiding reading of data that would be filtered by join
+condition.
 
-Consider the following query which captures a common pattern of a fact table ``store_sales``
-joined with a filtered dimension table ``date_dim``:
+Consider the following query which captures a common pattern of a fact table
+``store_sales`` joined with a filtered dimension table ``date_dim``:
 
     SELECT count(*)
     FROM store_sales
     JOIN date_dim ON store_sales.ss_sold_date_sk = date_dim.d_date_sk
     WHERE d_following_holiday='Y' AND d_year = 2000;
 
-Without dynamic filtering, Trino pushes predicates for the dimension table to the
-table scan on ``date_dim``, and it scans all the data in the fact table since there
-are no filters on ``store_sales`` in the query. The join operator ends up throwing away
-most of the probe-side rows as the join criteria is highly selective.
+Without dynamic filtering, Trino pushes predicates for the dimension table to
+the table scan on ``date_dim``, and it scans all the data in the fact table
+since there are no filters on ``store_sales`` in the query. The join operator
+ends up throwing away most of the probe-side rows as the join criteria is highly
+selective.
 
-When dynamic filtering is enabled, Trino collects candidate values for join condition
-from the processed dimension table on the right side of join. In the case of broadcast joins,
-the runtime predicates generated from this collection are pushed into the local table scan
-on the left side of the join running on the same worker.
+When dynamic filtering is enabled, Trino collects candidate values for join
+condition from the processed dimension table on the right side of join. In the
+case of broadcast joins, the runtime predicates generated from this collection
+are pushed into the local table scan on the left side of the join running on the
+same worker.
 
-Additionally, these runtime predicates are communicated to the coordinator over the network
-so that dynamic filtering can also be performed on the coordinator during enumeration of
-table scan splits.
+Dynamic filtering is enabled by default. To disable dynamic filtering, set the
+configuration property ``enable-dynamic-filtering=true`` to
```

Comment:
either
set the configuration property ``enable-dynamic-filtering=false``
or
set the configuration property ``enable-dynamic-filtering`` to ``false``

Reply:
"Dynamic filtering is enabled by default using the ``enable-dynamic-filtering`` configuration property. To disable dynamic filtering, set the configuration property to ``false``."

It would do this by default in the code, not in a config file where it is set to true out of the box, I believe.

Reply:
When something is enabled by default in the code, the user won't see any entry for it in the config file.
So I would write it as
```
Dynamic filtering is enabled by default. 
To disable dynamic filtering, set the configuration property ``enable-dynamic-filtering`` to ``false``. 
Alternatively, use the session property ``enable_dynamic_filtering``.
```